.github/scripts/utils.py/update-crowdin-locales.py

```diff
@@ -0,0 +1,88 @@
+import os
+import logging
+import json
+
+from crowdin_api import CrowdinClient
+from fnmatch import fnmatch
+
+CROWDIN_PROJECT_ID = 51028
+CROWDIN_API_TOKEN = os.environ["CROWDIN_PAT"]
+DEFAULT_LOCALE = "en-US"
+
+crowdin_api = CrowdinClient(token=CROWDIN_API_TOKEN, project_id=CROWDIN_PROJECT_ID, page_size=500)
+
+logging.basicConfig()
```

Comment:
Logger should be initialized in a `main` function ,ideally with support for a log level argument.